frontend/pages/index.js

```diff
@@ -1,56 +1,237 @@
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
+    <div>
+      {sourceLoading ? (
+        <LoadingScreen />
+      ) : (
+        <div>
+          {/* Start of Header */}
+          <div className={styles.contain}>
+            <div data-aos="fade" className={styles.headline}>
+              <h1>UNSW</h1>
+              <h1>Women In</h1>
+              <h1>Technology</h1>
+              <p> Empowering and inspiring the architects of change</p>
+              <button>
+                <Link href="/join-us">JOIN US</Link>
+              </button>
+            </div>
+          </div>
+          {/* End of Header */}
+
+          {/* Start of Description */}
+          <div
+            data-aos={isMobile ? "fade" : "fade-right"}
+            className={styles.description}
           >
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
+            <div className={styles.descriptionLeft}>
+              <h1>COLLABORATE. INSPIRE. CHANGE.</h1>
+            </div>
+            <div className={styles.descriptionRight}>
+              <p>
+                Formed in late 2016, we have grown to be a platform that
+                empowers, unites and up-skills female and male students alike
+                that are passionate about our mission. Women in Technology (WIT)
+                aims to cultivate future leaders that are prepared to challenge
+                the prejudices and bring change into the industry while building
+                a strong community where they can find lasting friendships and
+                support.
+              </p>
+            </div>
+          </div>
+          {/* End of Description */}
+          {/* Quotes */}
+          <div
+            data-aos={isMobile ? "fade" : "fade-left"}
+            className={styles.quotes}
+          >
+            <h1>HEAR FROM US</h1>
+            <div className={[styles.carousel, styles.quoteCarousel].join(" ")}>
+              <QuoteSlideshow
```

Comment:
Please test responsiveness for all devices

![image](https://user-images.githubusercontent.com/80564580/177753373-3cb339b6-5c7c-435b-a38c-44ac73a66ea2.png)